package.json

```diff
@@ -4,26 +4,25 @@
   "description": "amplify-cli",
   "scripts": {
     "addwords": "node ./scripts/add-to-dict.js",
-    "build-and-verify": "yarn clean && yarn dev-build && yarn prettier-check && yarn lint-check && yarn verify-api-extract",
-    "build-tests-changed": "nx affected --target=build-tests",
-    "build-tests": "nx run-many --target=build-tests --all",
-    "build": "nx run-many --target=build --all",
+    "build-and-verify": "yarn clean && yarn build && yarn prettier-check && yarn lint-check && yarn verify-api-extract",
+    "build-tests-changed": "lerna run build-tests --since dev",
+    "build-tests": "lerna run build-tests",
+    "build": "lerna run build",
     "check-version-conflicts:beta": "yarn ts-node ./scripts/check-version.ts beta",
     "check-version-conflicts:release": "yarn ts-node ./scripts/check-version.ts",
-    "clean": "nx reset && nx run-many --target=clean --all && yarn rimraf tsconfig.tsbuildinfo",
+    "clean": "lerna run clean && lerna exec yarn rimraf tsconfig.tsbuildinfo && lerna clean --yes && yarn rimraf node_modules",
```

Comment:
When I ran yarn clean, I got a Usage Error saying that it can't find a script named "rimraf"

Reply:
can you try running `yarn && yarn clean`, it looks that it cannot find the dependency that might need to be installed first

Reply:
I think its taking global rimraf for eddy. Should this be: `lerna exec yarn ./node_modules/bin/rimraf  tsconfig.tsbuildinfo`

Reply:
changing it to just `yarn rimraf` without lerna exec seems to fix it, will update